third_party/xla/xla/stream_executor/rocm/rocm_driver.cc

```diff
@@ -1967,12 +1967,17 @@ static tsl::StatusOr<T> GetSimpleAttribute(hipDevice_t device,
   // to allow for late allocations by internal ROCm libraries
   // (e.g. rocBLAS alone needs~200 MB to put its kernels as of ROCm 4.1)
   const uint64_t RESERVED_GFX908 = 1048576 * 512;
-  const uint64_t RESERVED_GFX9_X = 1048576 * 1024;
+  const uint64_t RESERVED_GFX90A = 1048576 * 1024;
+  // MI300 needs some more reserved GPU memory due to enablement of
```

Comment:
I think should be: MI300 (ROCm 6.1) ?

Reply:
Updated the comment.

Reply:
I mean since we already added the comment to indicate it, we can have the macro check for rocm version as well? But this is for [develop-upstream-QA-rocm61](https://github.com/ROCm/tensorflow-upstream/tree/develop-upstream-QA-rocm61), I think we can add this macro version check at develop-upstream branch.

Reply:
Oh, I agree. Let's do that when upstreaming to XLA, and then get it to develop-upstream via weekly sync.